### ISSUE TITLE
Add Boxing Gloves

### ISSUE THREAD

**Issue by MetaHatter** [InZidiuZ/op-framework-issue-tracker#2147]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes

**Summary:**
Write a summary of what your suggestion is about.
Add wearable boxing gloves as a weapon.

- 5% of actual fist damage and ignores armor.
- if Final blow is from gloves (or fists?) player is rendered unconscious and will stand back up after 30 seconds.
- Add very small chance for injury to keep EMS in the gameplay loop

**Reason:**
Why would your suggestion be a good change to the framework?
This would allow boxing training/matches/events and the like to take place in the city without criminal charges, a need for EMS oversight and avoid trips to pillbox.

Can tie into #2151

**Comment by Smolnisno** [comment]
I think this would be amazing for fight night events and similar types as well.

**Comment by InZidiuZ** [comment]
This has now been added
> 5% of actual fist damage and ignores armor.

I made it 25% of the fist as GTA deals with health with 100 integers only, making it hard for super small health changes. Fists also already ignore armor as it is.

> * if Final blow is from gloves (or fists?) player is rendered unconscious and will stand back up after 30 seconds.
> * Add very small chance for injury to keep EMS in the gameplay loop

These two requires slightly more complex logic with a lot of changes in deprecated systems & we also have the issue of someone taking most damage from other sources, then the final blow is a boxing glove. Would need some more consideration.

I am gonna close this for now and implement a more basic version of it with room for expansion in the future.
